app/models/pawn.rb

```diff
@@ -1,3 +1,44 @@
 class Pawn < Piece
 
+	def valid_move?(new_x, new_y)
+		[move_number_allowed?(new_y), no_vertical_obstacle?(new_y), not_a_backwards_move?(new_y), diagonal_move_allowed?(new_x, new_y)].all?
```

Comment:
Splitting line 4 into multiple lines like:
```
  def valid_move?(new_x, new_y)
    [move_number_allowed?(new_y),
     no_vertical_obstacle?(new_y),
     not_a_backwards_move?(new_y),
     diagonal_move_allowed?(new_x, new_y)].all?
  end
```
will make it a lot easier to read on Github or small screens (80 characters is the preferred line length according to the [Ruby style guide](https://github.com/styleguide/ruby)).